static/app/views/performance/transactionSummary/transactionEvents/transactionsTable.tsx

```diff
@@ -0,0 +1,236 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {Location, LocationDescriptor, Query} from 'history';
+
+import GuideAnchor from 'app/components/assistant/guideAnchor';
+import SortLink from 'app/components/gridEditable/sortLink';
+import Link from 'app/components/links/link';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import PanelTable from 'app/components/panels/panelTable';
+import Tooltip from 'app/components/tooltip';
+import {IconQuestion} from 'app/icons';
+import {t} from 'app/locale';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+import {Organization} from 'app/types';
+import {TableData, TableDataRow} from 'app/utils/discover/discoverQuery';
+import EventView, {MetaType} from 'app/utils/discover/eventView';
+import {getFieldRenderer} from 'app/utils/discover/fieldRenderers';
+import {Alignments, fieldAlignment, getAggregateAlias} from 'app/utils/discover/fields';
+import CellAction, {Actions} from 'app/views/eventsV2/table/cellAction';
+import {TableColumn} from 'app/views/eventsV2/table/types';
+import {GridCell, GridCellNumber} from 'app/views/performance/styles';
+import {TrendsDataEvents} from 'app/views/performance/trends/types';
+
+type Props = {
+  eventView: EventView;
+  organization: Organization;
+  location: Location;
+  isLoading: boolean;
+  tableData: TableData | TrendsDataEvents | null;
+  columnOrder: TableColumn<React.ReactText>[];
+  titles?: string[];
+  generateLink?: Record<
+    string,
+    (
+      organization: Organization,
+      tableRow: TableDataRow,
+      query: Query
+    ) => LocationDescriptor
+  >;
+  handleCellAction?: (
+    c: TableColumn<React.ReactText>
+  ) => (a: Actions, v: React.ReactText) => void;
+};
+
+class TransactionsTable extends React.PureComponent<Props> {
```

Comment:
Hey Shruthi, yes we do! There is an existing `TransactionsTable` nested in the `TransactionsList.tsx`. I chose to break it out into a new component to avoid conflict with existing usages (same as above) but I ended not having to diverge very much from the original `TransactionsTable` implementation. I'll take a stab at refactoring out the original `TransactionsTable` into it's own importable component and see if I can just reuse that. Thanks!

Reply:
Managed to breakout `TransactionsTable` from the `TransactionsList.tsx` file and use that instead.